src/chaplin/views/layout.coffee

```diff
@@ -107,30 +110,31 @@ module.exports = class Layout extends View
   openLink: (event) =>
     return if utils.modifierKeyPressed event
 
-    el = event.target
+    el = if Backbone.NativeView
```

Comment:
This shouldn't be relied on as a test. We may remove that behavior entirely at some point.

Much better is to test the converse, `Backbone.$`